faker/utils/loading.py

```diff
@@ -4,6 +4,13 @@
 
 def list_module(module):
     path = os.path.dirname(module.__file__)
+    try:
+        import pkgutil
+        modules = [name for finder, name, is_pkg in pkgutil.iter_modules([path]) if is_pkg]
+        if len(modules) > 0:
+            return modules
+    except (ImportError, AttributeError):
+        pass
```

Comment:
I'd rather avoid an `except ...: pass` pattern. Does `pkgutil.iter_modules` work for non-zip packages as well? If it does and it's available on all Python versions that we support, I'd rather just use that as the one single way to list modules.